awscli/examples/ssm/describe-instance-patches.rst

```diff
@@ -55,3 +55,30 @@ Output::
     ],
     "NextToken": "--token string truncated--"
   }
+
+**To get a list of patches installed patches since a specified InstalledTime for an instance**
+  
+This example combines the use of --filters and --query to retrieve information about patches installed since a specified time for the specified instance.
```

Comment:
It would be nice if we could follow similar wording to the other [examples](https://docs.aws.amazon.com/cli/latest/reference/ssm/describe-instance-patches.html#examples) in this file. The suggestion also appends `::` to follow rst formatting.

```suggestion
The following ``--describe-instance-patches`` example retrieves information about patches... ::
```

Reply:
Updated formatting to match the wording styles from other examples